Makefile

```diff
@@ -23,9 +23,12 @@ dependencies:
 cpan:
 	cpanm --notest --installdeps ./
 
-start: ## Start cors-proxy from build Docker image
+start: ## Start cors-proxy from build Docker image (run `make build-builder` or `make build-runtime` before)
 	docker run -p 8080:8080 --rm $(IMAGE_NAME)
 
+start-dev: ## Same as `start`, but using source code (including nginx conf and Lua) from local directory
+	docker run -p 8080:8080 --rm -v $(PWD):/opt/app-root/src $(IMAGE_NAME)	
```

Comment:
This will break if you installed `lua_modules` locally. Because it will override the ones in docker with your local ones. Might be good to add `rm -rf lua_modules` before this.

Reply:
Makes sense... And what about this?
```-v $(PWD)/src:/opt/app-root/src/src -v $(PWD)/nginx:/opt/app-root/src/nginx```

So we copy only the stuff we need. Not sure we'll need `spec` and `t`, I think not.

Reply:
I guess that could work too.